src/compiler/compile/nodes/Element.ts

```diff
@@ -93,6 +93,11 @@ const passive_events = new Set([
 	'touchcancel'
 ]);
 
+const react_attributes = new Map([
+	['className', 'class'],
```

Comment:
is `class` invalid?

also, it looks like the next line is failing `lint`

Reply:
this is `new Map([  [key, value] ])`, the key-value pair where `className` is invalid and should be replaced with `class`